app/views/govuk_publishing_components/components/_add_another.html.erb

```diff
@@ -0,0 +1,10 @@
+<%
+  add_button_text ||= "Add another"
+%>
```

Comment:
Would it be appropriate to apply the [wrapper helper](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component-wrapper-helper.md) to this component? I thought possibly not as this wrapper element isn't really particularly significant